Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +15,101 @@ namespace BootCamp.Chapter
+        {
+            string userName, userSurname;
+            userName = PromptString("Please input your name.");
+            userSurname = PromptString("Please input your surname.");
+
+            Console.WriteLine("Hi " + userName + " " + userSurname + "!");
+
+            int userAge;
+            userAge = PromptInt("Please input your age.");
+
+            float userWeight, userHeight;
+            userWeight = PromptFloat("Please input your weight in kg.");
+            userHeight = PromptFloat("Please input your height in meters.");
+
+            Console.WriteLine("Calculating BMI for " + userName + " " + userSurname + ".");
+            Console.WriteLine("Age: " + userAge);
+            Console.WriteLine("BMI: " + CalculateBmi(userWeight, userHeight) + "\n");
+        }
+
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            bool validInput = false;
+            int userInput = 0;
+
+            while (!validInput)
+            {
+                Console.WriteLine(message);
+                string consoleInput = Console.ReadLine();
+                validInput = Int32.TryParse(consoleInput, out userInput);
+
+                if (validInput)
+                {
+                    break;
+                }
+
+                Console.WriteLine("Invalid number.");
+            }
+
+            return userInput;
         }
 
         public static string PromptString(string message)
         {
-            // To do: call your implementation. 
-            return "";
+            bool validInput = false;
+            string userInput = "";
+
+            while (!validInput)
+            {
+                Console.WriteLine(message);
+                userInput = Console.ReadLine().Trim();
+                validInput = userInput.Length > 0;
+
+                if (validInput)
+                {
+                    break;
+                }
+
+                Console.WriteLine("Empty string.");
+            }
+
+            return userInput;
         }
 
         public static float PromptFloat(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            bool validInput = false;
+            float userInput = 0f;
+
+            while (!validInput)
+            {
+                Console.WriteLine(message);
+                string consoleInput = Console.ReadLine();
+                validInput = float.TryParse(consoleInput, NumberStyles.Float, CultureInfo.InvariantCulture, out userInput);
+
+                if (validInput)
+                {
+                    break;
+                }
+
+                Console.WriteLine("Invalid number.");
+            }
+
+            return userInput;
         }
 
         public static float CalculateBmi(float weight, float height)
         {
-            // To do: call your implementation. 
-            return 0;
+            if (height == 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not check floating point equality with exact values, use a range instead.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4868137)